docs/data/material/components/stacked-snackbars/stacked-snackbars.md

```diff
@@ -0,0 +1,91 @@
+---
+product: material-ui
+title: Stacked snackbars React component
+components: SnackbarsProvider
+materialDesign: https://material.io/components/snackbars
+waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
+---
+
+# Stacked snackbars
+
+<p class="description">The <code>SnackbarsProvider</code> component and <code>useSnackbars</code> hook allow you to display multiple stacked snackbars.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+:::info
+ℹ️ Note that Material Design doesn't advocate for this behavior, as seen in [their snackbar documentation](https://material.io/components/snackbars#usage).
+However, given this is a common pattern seen in multiple webapps, Material UI allows deisgners and developers to break away from that guideline.
+:::
+
+## Usage
+
+To display multiple snackbars, wrap your application with the `SnackbarsProvider` component.
+That's what will enable you to use the `useSnackbars` hook.
+
+If you're using `ThemeProvider`, wrap `SnackbarsProvider` below it, as follows:
+
+```jsx
+<ThemeProvider>
+  <SnackbarsProvider>
+    <YourApp />
+  </SnackbarsProvider>
+</ThemeProvider>
+```
+
+## Maximum snackbars
+
+The `limit` prop can be passed to `SnackbarsProvider` to limit the maximum number of snackbars displayed at the same time.
+
+{{"demo": "MaximumSnackbars.js", "defaultCodeOpen": false}}
+
+## Basic stacked snackbars
+
+The `message` prop is used to show the snackbar message.
+
+The `action` prop can be used to show content on the right side of the snackbar.
+It can simply be a React Node or a callback function with a key parameter returning a React Node.
+
+Use the `close` method to dismiss a specific snackbar by providing the `key` from the `action` callback argument.
+
+{{"demo": "SimpleSnackbars.js", "defaultCodeOpen": false}}
+
+## Customization
+
+Use the `content` prop to customize the individual snackbars.
```

Comment:
Lead into the demo.

```suggestion
Use the `content` prop to customize individual snackbars, as shown in this demo:
```

Reply:
Customize how, exactly? Is this what makes it possible to add custom components (in this case the `<Alert>`) inside of an individual snackbar with this prop? I think we need to be more specific here.

Reply:
Yes, we can add any custom components, like here it's `<Alert />`. How about this:
```suggestion
Use the `content` prop to customize the individual snackbars. You can build your own custom component and render it by passing it to the `content` prop. 
The following demo renders the `Alert` component in the snackbar by passing it to the `content` prop:
```